tests/perform_login.spec.js

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from "@playwright/test";
+import { constants } from "../constants";
+import PageElements from "../pages/page_elements";
+
+require("dotenv").config({
+  path: "/Users/eugene_minikh/Work_project/Yajsc_PW/YAJSC_PW/.env",
+});
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(constants.baseUrl);
```

Comment:
You can move this to the config.

```
baseURL: 'constants.baseUrl',
```

Here - https://github.com/EugeneMinikh/YAJSC_PW/pull/3/files#diff-8421b2f4ad242eb670ea84ae0057d8708f0370adad5ac063da20b8abc54426daR28

After that you can call the method as 

```
await page.goto('/');
```